helm/vertical-pod-autoscaler-app/values.yaml

```diff
@@ -1,189 +1,85 @@
+        memory: 3000Mi
+      requests:
+        cpu: 50m
+        memory: 500Mi
+
+    extraArgs:
+      pod-recommendation-min-cpu-millicores: 15
+      pod-recommendation-min-memory-mb: 100
+      v: 2
+
+    metrics:
+      serviceMonitor:
+        # Specifies whether a service monitor should be created
+        enabled: false
+
+  updater:
+    # Enable the component
     enabled: true
+
+    replicaCount: 1
+
     image:
-      # admissionController.cleanupOnDelete.image.repository -- The repository of the post-delete image
-      repository: giantswarm/vpa-certgen-ci-images
-      # admissionController.cleanupOnDelete.image.tag -- The image tag to use for the admission controller cleanup image
-      tag: v11-alpine-openssl
-  # admissionController.replicaCount -- The amount admission-controller pods which should run
-  replicaCount: 2
-  # admissionController.podDisruptionBudget -- This is the setting for the pod disruption budget
-  podDisruptionBudget:
-    # maxUnavailable: 1
-    minAvailable: 1
-  image:
-    # admissionController.image.repository -- The location of the vpa admission controller image
-    repository: giantswarm/vpa-admission-controller
-    # admissionController.image.pullPolicy -- The pull policy for the admission controller image. Recommend not changing this
-    pullPolicy: IfNotPresent
-    # admissionController.image.tag -- Overrides the image tag whose default is the chart appVersion
-    tag: ""
-  # admissionController.podAnnotations -- Annotations to add to the admission controller pod
-  podAnnotations: {}
-  # admissionController.podLabels -- Labels to add to the admission controller pod
-  podLabels: {}
-  # admissionController.podSecurityContetxt -- The security context for the admission controller pod
-  podSecurityContext:
-    runAsNonRoot: true
-    runAsUser: 65534
-  # admissionController.securityContext -- The security context for the containers inside the admission controller pod
-  securityContext: {}
-  # admissionController.resources -- The resources block for the admission controller pod
-  resources:
-    limits:
-      cpu: 200m
-      memory: 2000Mi
-    requests:
-      cpu: 50m
-      memory: 200Mi
-  # admissionController.tlsSecretKeys -- The keys in the vpa-tls-certs secret to map in to the admission controller
-  tlsSecretKeys: []
-    # - key: ca.crt
-    #   path: caCert.pem
-    # - key: tls.crt
-    #   path: serverCert.pem
-    # - key: tls.key
-    #   path: serverKey.pem
-  nodeSelector: {}
-  tolerations: []
-  affinity: {}
-
-isManagementCluster: false
-registry:
-  domain: docker.io
+      repository: docker.io/giantswarm/vpa-updater
+      tag: 0.11.0
+      pullPolicy: IfNotPresent
+
+    resources:
+      limits:
+        cpu: 200m
+        memory: 3000Mi
+      requests:
+        cpu: 50m
+        memory: 500Mi
+
+    extraArgs:
+      v: 2
+
+    metrics:
+      serviceMonitor:
+        # Specifies whether a service monitor should be created
+        enabled: false
```

Comment:
I would enable service monitors though

Reply:
Are we ready for servicemonitors on all WCs?
I thought we had to keep relying on the `giantswarm.io/monitoring` label for apps that will be deployed on WCs as long as some WCs don't run prometheus-agent.

Reply:
If we only release this in aws 18.2+ and azure 19+ then yes

Reply:
There's no earlier WC that depend on collections?

Reply:
Collections? No but we need to make sure phoenix does not release that in older releases (i.e. fix on 17.x)